src/main/java/org/broadinstitute/hellbender/engine/dataflow/DataflowCommandLineProgram.java

```diff
@@ -25,6 +26,8 @@
 public abstract class DataflowCommandLineProgram extends CommandLineProgram implements Serializable {
     private static final long serialVersionUID = 1l;
 
+    public interface HellbenderDataflowOptions extends GCSOptions, DataflowPipelineOptions {}
```

Comment:
Because GCSOptions changed. It used to include everything we needed, now some of that stuff is moved onto DataflowPipelineOptions. And since neither inherits from the other, we need to ask for both in order to get the options we need.